zeek/zeek.spec

```diff
@@ -45,7 +45,7 @@ Requires:         caf >= %{CAF_VER}
 BuildRequires:    caf-devel >= %{CAF_VER}
 Requires:         bind-libs
 BuildRequires:    bind-devel
-Requires:         libmaxminddb
+Requires:         libmaxminddb0
```

Comment:
According to https://src.fedoraproject.org/rpms/libmaxminddb/blob/epel7/f/libmaxminddb.spec#_1, which is the upstream source to the package provided by EPEL, the package name is `libmaxminddb`.

Reply:
Without this change I am getting 
```
Transaction check error:
  file /usr/lib64/libmaxminddb.so.0.0.7 from install of libmaxminddb-1.2.0-6.el7.x86_64 conflicts with file from package libmaxminddb0-1.3.2-1.el7.x86_64
```
on CentOS and RedHat

Reply:
This repository is building libmaxmindb0 I assumed that is the one you are attempting to install, is the intent to not build the version in this repository and use it?

Reply:
Sorry for the long turn around here. I like to use upstream packages when possible. We did build the libmaxminddb library for a bit until the newer version was included in EPEL7. The EPEL version is now `libmaxminddb`.